keyboards/ymdk/oni_hhkb/keyboard.json

```diff
@@ -0,0 +1,236 @@
+            {"matrix": [2, 3], "x": 60, "y": 32, "flags": 4},
+            {"matrix": [2, 4], "x": 76, "y": 32, "flags": 4},
+            {"matrix": [2, 5], "x": 92, "y": 32, "flags": 4},
+            {"matrix": [2, 6], "x": 108, "y": 32, "flags": 4},
+            {"matrix": [2, 7], "x": 124, "y": 32, "flags": 4},
+            {"matrix": [2, 8], "x": 140, "y": 32, "flags": 4},
+            {"matrix": [2, 9], "x": 156, "y": 32, "flags": 4},
+            {"matrix": [2, 10], "x": 172, "y": 32, "flags": 4},
+            {"matrix": [2, 11], "x": 188, "y": 32, "flags": 4},
+            {"matrix": [2, 13], "x": 214, "y": 32, "flags": 1},
+            {"matrix": [3, 0], "x": 10, "y": 48, "flags": 1},
+            {"matrix": [3, 2], "x": 36, "y": 48, "flags": 4},
+            {"matrix": [3, 3], "x": 52, "y": 48, "flags": 4},
+            {"matrix": [3, 4], "x": 68, "y": 48, "flags": 4},
+            {"matrix": [3, 5], "x": 84, "y": 48, "flags": 4},
+            {"matrix": [3, 6], "x": 100, "y": 48, "flags": 4},
+            {"matrix": [3, 7], "x": 116, "y": 48, "flags": 4},
+            {"matrix": [3, 8], "x": 132, "y": 48, "flags": 4},
+            {"matrix": [3, 9], "x": 148, "y": 48, "flags": 4},
+            {"matrix": [3, 10], "x": 164, "y": 48, "flags": 4},
+            {"matrix": [3, 11], "x": 180, "y": 48, "flags": 4},
+            {"matrix": [3, 13], "x": 202, "y": 48, "flags": 1},
+            {"matrix": [3, 14], "x": 224, "y": 48, "flags": 4},
+            {"matrix": [4, 0], "x": 4, "y": 64, "flags": 1},
+            {"matrix": [4, 1], "x": 24, "y": 64, "flags": 1},
+            {"matrix": [4, 2], "x": 44, "y": 64, "flags": 1},
+            {"matrix": [4, 7], "x": 112, "y": 64, "flags": 4},
+            {"matrix": [4, 10], "x": 180, "y": 64, "flags": 1},
+            {"matrix": [4, 12], "x": 200, "y": 64, "flags": 1},
+            {"matrix": [4, 13], "x": 220, "y": 64, "flags": 1},
+            {"x": 8, "y": 4, "flags": 2},
+            {"x": 48, "y": 4, "flags": 2},
+            {"x": 88, "y": 4, "flags": 2},
+            {"x": 132, "y": 4, "flags": 2},
+            {"x": 180, "y": 4, "flags": 2},
+            {"x": 216, "y": 4, "flags": 2},
+            {"x": 8, "y": 52, "flags": 2},
+            {"x": 48, "y": 52, "flags": 2},
+            {"x": 96, "y": 52, "flags": 2},
+            {"x": 132, "y": 52, "flags": 2},
+            {"x": 180, "y": 52, "flags": 2},
+            {"x": 216, "y": 52, "flags": 2}
+        ],
+        "max_brightness": 200
+    },
+    "url": "https://ymdkey.com/products/ymdk-gh60-hhkb-style-tsangan-oni-hot-swappable-both-underglow-switch-rgb-via-programmable-pcb-compatible-with-tokyo-60",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0021",
+        "vid": "0x45D4"
+    },
+    "ws2812": {
+        "pin": "E2"
+    },
+    "community_layouts": ["60_ansi_tsangan_split_bs_rshift"],
+    "layout_aliases": {
+        "LAYOUT_60_tsangan_hhkb": "LAYOUT_60_ansi_tsangan_split_bs_rshift"
+    },
```

Comment:
Thankfully this PR is a keyboard addition rather than changes to an existing keyboard so layout alias isn't required
```suggestion
```

Reply:
It wouldn't be required, but 60_tsangan_hhkb is a (quasi?) standard for many boards that feature the split bs and shift. I use this layout name for all my HHKB boards, so I'm a bit reluctant to remove the alias in this case. Wdyt?